docs/source/envisage_core_documentation/extension_points.rst

```diff
@@ -56,31 +56,7 @@ point that allows other plugins to contribute new messages. Using the
 
 	...
 
-Overriding the 'get_extension_points' method might look somthing like::
```

Comment:
If we're deleting this should we remove the "2) Programmatically - by overriding the 'get_extension_points' method" above?